packages/frontend/src/pages/bridges-risk/BridgesRiskView.tsx

```diff
@@ -35,9 +42,19 @@ export function BridgesRiskView({ items }: BridgesRiskViewProps) {
       minimalWidth: true,
       getValue: (entry, index) => (
         <>
-          <span data-bridges-only>{onlyBridges.indexOf(entry) + 1}</span>
+          <span data-bridges-only>
+            {entry.isVerified === false ? (
+              <UnverifiedWarning message="This project includes unverified contracts" />
+            ) : (
+              onlyBridges.indexOf(entry) + 1
+            )}
```

Comment:
This is being repeated over and over. There could be a separate component `components/table/IndexCell.tsx`, with `interface IndexCellProps { index: number, isVerified: boolean }` and would encapsulate this ternary operator.